_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: otel_logs_source
+parent: Sources
+grand_parent: Pipelines
+nav_order: 25
+---
+
+# otel_logs_source
+
+## Overview
+
+`otel_logs_source` is an OpenTelemetry source that follows the [OpenTelemtry Protocol Specification](https://github.com/open-telemetry/oteps/blob/master/text/0035-opentelemetry-protocol.md), receives logs from OpenTelemetry Collector, and receives `ExportLogsServiceRequest` records to process through a Data Prepper pipeline.
+
+This source supports the `OTLP/gRPC` protocol.
+{: .note}
+
+## Configuration
+
+You can configure the `otel_logs_source` source with the following options. 
+
+| Option | Type | Description |
+| :--- | :--- | :--- |
+| port | int | Represents the port that the `OTel logs source` is running on. Default value is `21892`. |
+| path | string | Represents the path for sending unframed HTTP requests. This can be used for supporting unframed gRPC with HTTP idiomatic path to a configurable path. It should start with `/` and length should be at least 1. `/opentelemetry.proto.collector.logs.v1.LogsService/Export` endpoint will be disabled for both gRPC and HTTP requests if path is configured. Path can contain `${pipelineName}` placeholder which will be replaced with pipeline name.  If the value is empty and `unframed_requests` is `true`, then the path that the source provides is `/opentelemetry.proto.collector.logs.v1.LogsService/Export`. There is no default value. | 
+| request_timeout | int | Represents the request timeout duration in milliseconds. Default value is `10,000`. |
```

Comment:
As far as I know YAML does not support commas in numbers. 

So while the default value is 10,000, it would need to be represented as `10000` in the actual configuration. So I think it might be best to include this in the actual value.

But, I could see an argument for making it more readable in the docs. Perhaps removing the backticks?

Thoughts?

Reply:
@natebower Do you have any specific recommendations for removing the tic marks? All default values in this table are inside tic marks.

Reply:
Leave in tic marks but render as `10000` (if that's how it appears to the user).